tests/test_core.py

```diff
@@ -3374,6 +3386,47 @@ def indir(name):
       '''
     self.do_run(src, 'a: loaded\nb: loaded\na: loaded\n')
 
+  @needs_dlfcn
+  @bleeding_edge_wasm_backend
+  def test_dlfcn_feature_in_lib(self, js_engines):
+    self.emcc_args.append('-mnontrapping-fptoint')
+
+    self.prep_dlfcn_lib()
+    lib_src = r'''
+        extern "C" int magic(float x) {
+          return __builtin_wasm_trunc_saturate_s_i32_f32(x);
+        }
+      '''
+    dirname = self.get_dir()
+    filename = os.path.join(dirname, 'liblib.cpp')
+    self.set_setting('EXPORTED_FUNCTIONS', ['_splat_inc_and_get'])
```

Comment:
Where does `_splat_inc_and_get` come from?

Reply:
Ah.. that should be "magic" after I renamed it. But the line is not needed anyhow, there isn't a need to export functions in a side module (we don't dce shared modules by default). Will fix, thanks.